docusaurus/docs/shared/set-up-environment.md

```diff
@@ -13,18 +13,28 @@ To discover plugins, Grafana scans a _plugin directory_, the location of which d
 
 1. Restart Grafana if it's already running, to load the new configuration.
 
-### Alternative method: Docker
+### Alternative method: Docker compose
 
-If you don't want to install Grafana on your local machine, you can use [Docker](https://www.docker.com).
+If you don't want to install Grafana on your local machine, you can use [Docker Compose](https://docs.docker.com/compose).
 
-To set up Grafana for plugin development using Docker, run the following command:
+The template create by `@grafana/create-plugin` contains a `docker-compose.yaml` file with everything necessary and you just have to run:
 
 ```
-docker run -d -p 3000:3000 -v "$(pwd)"/grafana-plugins:/var/lib/grafana/plugins --name=grafana grafana/grafana:7.0.0
+docker compose up -d
```

Comment:
the help text in `create-plugin` uses `docker-compose up` if theres no reason to pick one over the other, suggest making the change here as simpler.

Reply:
resolved in https://github.com/grafana/plugin-tools/pull/373/commits/2e8d7a87089941d1275c624ab527ebb59115c0b4